<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pfWindow</class>
 <widget class="QMainWindow" name="pfWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pfWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>7</y>
      <width>339</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="ipAddressLabel">
           <property name="text">
            <string>IP Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="serverIPLineEdit">
           <property name="text">
            <string>192.168.0.5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Port: </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>69</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="portLineEdit">
           <property name="text">
            <string>2000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="connectedLabel">
           <property name="text">
            <string>Connected:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="connectionStatusLabel">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="connectToServerButton">
         <property name="text">
          <string>Connect To Server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectFromServerButton">
         <property name="text">
          <string>Disconnect From Server</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>11</y>
      <width>791</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="startSensingButton">
       <property name="text">
        <string>Start Reading Sensors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberOfSensorValuesReceivedLabel">
       <property name="text">
        <string>Number of Sensor Values Received</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="IR_0Label">
         <property name="text">
          <string>IR_0:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_0CountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_1Label">
         <property name="text">
          <string>IR_1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_1CountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_2Label">
         <property name="text">
          <string>IR_2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_2CountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_3label">
         <property name="text">
          <string>IR_3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_3CountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_4label">
         <property name="text">
          <string>IR_4:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IR_4CountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="GrabberIsAlignedLabel">
         <property name="text">
          <string>Grabber is Aligned:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Grabber_AlignedLabel">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>1131</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IR_0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="IR_0PlainTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="IR_0_boolLabel">
           <property name="text">
            <string>IR_0_bool</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="IR_0_bool_textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>IR_1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="IR_1PlainTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="IR_1_boolLabel">
           <property name="text">
            <string>IR_1_bool</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="IR_1_bool_textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>IR_2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="IR_2PlainTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="IR_2_boolLabel">
           <property name="text">
            <string>IR_2_bool</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="IR_2_bool_textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="IR_3collabel">
           <property name="text">
            <string>IR_3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="IR_3PlainTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="IR_3label_2">
           <property name="text">
            <string>IR_3_bool</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="IR_3_bool_textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QLabel" name="IR_4collabel">
           <property name="text">
            <string>IR_4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="IR_4PlainTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QLabel" name="IR_3label_3">
           <property name="text">
            <string>IR_4_bool</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="IR_4_bool_textEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="AlignmentInfoLabel">
             <property name="text">
              <string>Alignment Information:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Alignment_InfoLabel">
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QPushButton" name="proximityPushButton">
               <property name="text">
                <string>Send Proximity</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="proxsentboollabel">
                 <property name="text">
                  <string>Proximity Sent Bool:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="prox_sent_bool_label">
                 <property name="text">
                  <string>N/A </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="proximityValueLabel">
               <property name="text">
                <string>Proximity Value:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="proximitySlider">
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
               <property name="sliderPosition">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="proxlcdNumber">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="intValue" stdset="0">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QPushButton" name="sendMovementPushButton">
             <property name="text">
              <string>Send Movement</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="movementBoollabel">
               <property name="text">
                <string>Movement Bool:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="movement_bool_label">
               <property name="text">
                <string>N/A</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QPushButton" name="sendOutsideofArenaPushButton">
             <property name="text">
              <string>Send Outside of Arena</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="outsideofArenaBoollabel">
               <property name="text">
                <string>Outside of Arena Bool:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="outside_of_arena_bool_label">
               <property name="text">
                <string>N/A</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QPushButton" name="requestTargetAcquiredpushButton">
             <property name="text">
              <string>Request Target Acquired</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="targetAcquiredlabel">
                 <property name="text">
                  <string>Target Acquired:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="target_acquired_bool_label">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="targetAcquiredReqSentBoollabel">
                 <property name="text">
                  <string>Target Acquired Request Sent:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="target_Acquired_Request_Sent_Bool_label">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>1131</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="expectedDegreeLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Expected Degree(s) to Turn:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="angleLcdNumber">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="DistanceFromTargetLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Distance from center of Rover to Target:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="distanceFromTargetLcdNumber"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculate_Variance">
    <property name="title">
     <string>Additional Features</string>
    </property>
    <addaction name="actionCalculate_Variance"/>
    <addaction name="actionRequests_Responses"/>
    <addaction name="actionTG_Requests_and_Responses"/>
   </widget>
   <addaction name="menuCalculate_Variance"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCalculate_Variance">
   <property name="text">
    <string>Calculate Variance</string>
   </property>
  </action>
  <action name="actionRequests_Responses">
   <property name="text">
    <string>PF Requests and Responses</string>
   </property>
  </action>
  <action name="actionTG_Requests_and_Responses">
   <property name="text">
    <string>TG Requests and Responses</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
